lib/Sema/CSDiagnostics.cpp

```diff
@@ -5862,12 +5862,18 @@ bool OutOfOrderArgumentFailure::diagnoseAsError() {
 
     // Move requires postfix comma only if argument is moved in-between
     // other arguments.
-    bool requiresComma =
-        !isExpr<BinaryExpr>(anchor) && PrevArgIdx != args->size() - 1;
+    auto sign = "";
+    if (isExpr<BinaryExpr>(anchor)) {
+      auto BinExpr = castToExpr<BinaryExpr>(anchor);
+      auto Func = BinExpr->getFn();
+      auto operatorName = std::string(*getOperatorName(Func));
+      sign = (" " + operatorName + " ").c_str();
+    } else if (PrevArgIdx != args->size() - 1) {
+      sign = ", ";
+    }
```

Comment:
Nit: A small simplification suggestion
```suggestion
    std::string sign;
    if (auto *BE = getAsExpr<BinaryExpr>(anchor)) {
      auto operatorName = std::string(*getOperatorName(BE->getFn()));
      sign = " " + operatorName + " ";
    } else if (PrevArgIdx != args->size() - 1) {
      sign = ", ";
    }
```

Reply:
Thanks for your code review and suggestions.  I've made the changes and unit tests have passed in my local.